compiler/internal/codegen/api_handlers.go

```diff
@@ -665,8 +665,7 @@ func (b *rpcBuilder) renderCaller() *Statement {
 			}
 			g.Err()
 		}).Op(":=").Id(b.rpcHandlerName(rpc)).Dot("Call").CallFunc(func(g *Group) {
-			// TODO implement
-			g.Qual("encore.dev/appruntime/api", "CallContext").Values()
+			g.Qual("encore.dev/appruntime/api", "NewCallContext").Call(Id("ctx"))
```

Comment:
By using `ctx` as a parent this change could result in values from the parent context being visible to children - which would not be the case when the request goes over an HTTP connection.

For this reason I think we should make the CallContext implement the `Values` interface, which doesn't recurse to the parent and returns nil - that way timeouts and context cancels properate down to the parent context - but the values do not (which would be the same behaviour if the API was called over the wire)